### PR TITLE
Intel Habanalabs bootc image

### PR DESCRIPTION
**Pull request by kwozyman**
Add support for Intel Habana Labs, based off RHEL 9.2 drivers for now.

### CODE REVIEW

File: training/README.md [hunk 1]
```diff
@@ -5,10 +5,11 @@ In order to run accelerated AI workloads, we've prepared [bootc](https://github.
...
 
-| Target     | Description                                   |
-|------------|-----------------------------------------------|
-| amd        | Create bootable container for AMD platform    |
-| nvidia     | Create bootable container for NVidia platform |
+| Target      | Description                                             |
+|-------------|---------------------------------------------------------|
+| amd         | Create bootable container for AMD platform              |
+| nvidia      | Create bootable container for NVidia platform           |
+| intel-gaudi | Create bootable container for Intel Habanalabs platform |
```
Comment (rhatdan) [inline training/README.md]: Do you think there will be other intel-nogaudi repos?  I would prefer to stick with just intel.

File: training/intel-gaudi/Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM ?= 
+
+REGISTRY ?= quay.io
+REGISTRY_ORG ?= ai-lab
+IMAGE_NAME ?= intel-gaudi
```
Comment (rhatdan) [inline training/intel-gaudi/Makefile]: intel-gaudi-bootc
or 
intel-bootc

File: training/README.md (hunk 1 above)
Comment (kwozyman) [reply training/README.md]: I'm not sure about the branding to be honest. I know there are some consumer Intel GPUs, but I don't know if those will ever be used for AI or not

File: training/intel-gaudi/Makefile (hunk 2 above)
Comment (kwozyman) [reply training/intel-gaudi/Makefile]: I'll say `intel-gaudi-bootc`, just based off my previous comment ^^

File: training/README.md (hunk 1 above)
Comment (rhatdan) [reply training/README.md]: I think the same could be said for nvidia and amd GPUs.  I think we should stick to intel, and then if we add some other intel gpus that we want to support but have to do it with other images, then we add intel-OTHERGPU.  With a strong preference for not doing this, and having Intel support all of their GPUs from a RHEL perspective.

Comment (kwozyman) [reply training/README.md]: Makes sense to me. Should I also rename the directory to just `intel`?

Comment (rhatdan) [reply training/README.md]: Yes

Comment (kwozyman) [reply training/README.md]: Done

**Comment by rhatdan** [comment]
LGTM

File: training/README.md (hunk 1 above)
Comment (airlied) [reply training/README.md]: intel will have a oneapi variant which is unrelated to this stack, it won't make sense to make a container with both gaudi and oneAPI supported devices (like discrete GPUs).

**Comment by airlied** [comment]
Does this leave out installing the synapseAI stack on purpose? seems like it should at least be mentioned (even if commented out).

**Comment by rhatdan** [comment]
I think we want a default, and over time we will add more variants although I would prefer if we could differentiate at runtime rather then having lots of images.

**Pull request merged**